packages/paste-core/components/minimizable-dialog/src/StyledMinimizableDialog.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+
+export const StyledMinimizableDialog: React.FC<{element?: string; children: React.ReactNode}> = ({
```

Comment:
issue(blocker): This needs to be changed for React18. We can no longer do `React.FC` typings without `React.PropsWithChildren`

```
React.FC<React.PropsWithChildren<AvatarContentProps>>
```

Reply:
Side note, worries me it doesn't fail type checking

Reply:
ooohh is this a react 18 thing?

Reply:
Yup Yup. I have updated the VS code snippet too, but I don't know if anyone uses them